test/batch_test.rb

```diff
@@ -8,6 +8,29 @@ def test_batch
     assert_equal 'Shirt', product.title
   end
 
+  class MyCustomExecutor < GraphQL::Batch::Executor
```

Comment:
I would prefer if you moved the new test to GraphQL::Batch::CustomExecutorTest so that we can use the MyCustomExecutor in there rather than copying it here.

Reply:
👍 moved it there